src/main/resources/swagger.api/schemas/cell.json

```diff
@@ -15,6 +15,11 @@
       "description": "Column visibility",
       "type": "boolean",
       "default": "true"
+    },
+    "ignoreTranslation": {
```

Comment:
API is changed. Please, increase minor version in corresponding interface in the ModuleDescriptor.

Reply:
@khandramai  was updated.